website/static/index.html

```diff
@@ -150,7 +150,7 @@
       <a href="#"><img class="logo" src="img/logo.svg" alt="All Contributors logo"/></a>
       <nav>
         <ul>
-          <li><a href="/docs">Docs</a></li>
+          <li><a href="/docs/en/overview/">Docs</a></li>
```

Comment:
We might not want this, it could force the English page when another language would be better. I _think_ Netlify is handling the redirection here [like this](https://docs.netlify.com/manage/routing/redirects/redirect-options/#redirect-by-country-or-language).

Reply:
Thanks @JimMadge --- the translations are the piece I just don't understand yet. Felipe is working on them separately which is awesome. And we are working on moving to astro for docs as well to simplify. But how translations actually work is the piece i'm most worried about as I don't get it. Thank you for this link - i'll read more about how it works!! (and now that netlify is working again i'll see if our previews work too!!

Reply:
so note here - https://deploy-preview-871--allcontributors-org.netlify.app/docs/de/overview the translations don't work but the link tot he english version does work. That magic not happening is what i'm worried about! Maybe it's ok tho given we have a plan to migrate everything?

Reply:
> @JimMadge yes! It looks like German exists. Would you like access to Crowdin as well?

Yes please! I would be great to get a feeling for how it works.

> I wonder what happens if we build the translations here in this repo and add them to version control?

I'm not certain but I guess that is also an option. It looks like the existing build process is 

1. fetch translations from Crowdin to your local filesystem
2. build the complete site

but it might be possible to instead have _all_ of the languages in the repo and have crowdin sync there 🤔. I need to find someone who knows Crowdin to talk me through it 😅.